cmake/findDependencies.cmake

```diff
@@ -38,3 +38,14 @@ if (NOT USE_MATCHCOMPILER_OPT MATCHES "Off")
         set(USE_MATCHCOMPILER_OPT "Off")
     endif()
 endif()
+
+if (NOT USE_BUNDLED_TINYXML2)
+    find_package(tinyxml2 QUIET)
```

Comment:
This should fail if it wasn't found. If you turn it off you expect the system one to be used so not having one should be an error.

If you enable Z3 or PCRE and it is missing we also don't turn it off again.

Reply:
@firewave, thanks for the feedback. I have added a commit now, which makes find_package(tinyxml2) fatal now, if it is not found.